ilastik/applets/tracking/conservation/opConservationTracking.py

```diff
@@ -132,84 +125,79 @@ def setupOutputs(self):
         # FIXME: assumes t,x,y,z,c
         chunks[0] = 1  # 't'        
         self._blockshape = tuple(chunks)
-        self._opCache.BlockShape.setValue(self._blockshape)
+        self._opCache.outerBlockShape.setValue(self._blockshape)
 
         self.AllBlocks.meta.shape = (1,)
         self.AllBlocks.meta.dtype = object
         
         self.MergerOutput.meta.assignFrom(self.LabelImage.meta)
         self.RelabeledImage.meta.assignFrom(self.LabelImage.meta)
-        self._ndim = 2 if self.LabelImage.meta.shape[3] == 1 else 3
 
-        self._mergerOpCache.BlockShape.setValue( self._blockshape )
-        self._relabeledOpCache.BlockShape.setValue( self._blockshape )
+        self._mergerOpCache.outerBlockShape.setValue( self._blockshape )
+        self._relabeledOpCache.outerBlockShape.setValue( self._blockshape )
         
         frame_shape = (1,) + self.LabelImage.meta.shape[1:] # assumes t,x,y,z,c order
         assert frame_shape[-1] == 1
         self.MergerOutput.meta.ideal_blockshape = frame_shape
         self.RelabeledImage.meta.ideal_blockshape = frame_shape
-        
-    
+          
     def execute(self, slot, subindex, roi, result):
+        # Output showing lineage IDs
         if slot is self.Output:
             if not self.Parameters.ready():
                 raise Exception("Parameter slot is not ready")
-            parameters = self.Parameters.value           
-            trange = range(roi.start[0], roi.stop[0])
+            parameters = self.Parameters.value
             
-            original = np.zeros(result.shape, dtype=slot.meta.dtype)         
+            # Assume [t,x,y,z,c] order           
+            trange = range(roi.start[0], roi.stop[0])
+            offset = roi.start[1:-1]
+       
             result[:] =  self.LabelImage.get(roi).wait()
 
             for t in trange:
-                if (self.mergerResolver 
-                        and 'time_range' in parameters 
-                        and t <= parameters['time_range'][-1] 
-                        and t >= parameters['time_range'][0]):
-                    self.mergerResolver.relabelMergers(result[t-roi.start[0],...,0], t)
+                if 'time_range' in parameters and t <= parameters['time_range'][-1] and t >= parameters['time_range'][0]:
+                    if self.ResolvedMergers.value:
+                        self._labelMergers(result[t-roi.start[0],...,0], t, offset)
```

Comment:
It might be better to pre-save the result of `self.ResolvedMergers.value` outside of this loop, since `.value` is probably relatively slow compared to normal property.  (Just a note for future reference; no need to change this PR if you'd rather not.)